src/Core/OOProbabilisticTextureManager.m

```diff
@@ -79,14 +79,14 @@ - (id)initWithPListName:(NSString *)plistName
 		_textures = malloc(sizeof *_textures * count);
 		_prob = malloc(sizeof *_prob * count);
 		_galaxy = malloc(sizeof *_galaxy * count);
-		_probMaxGal = malloc(sizeof *_probMaxGal * 8);
+		_probMaxGal = malloc(sizeof *_probMaxGal * kOOMaximumGalaxyID);
```

Comment:
I think this should be `_probMaxGal = malloc(sizeof *_probMaxGal * (kOOMaximumGalaxyID + 1));`